source/Common.h

```diff
@@ -140,7 +140,7 @@ enum AppMode_e
 #define WM_USER_FULLSCREEN	WM_USER+8
 #define VK_SNAPSHOT_TEXT	WM_USER+9 // PrintScreen+Ctrl
 
-#ifdef _MSC_VER
+#ifdef _WIN32
```

Comment:
`_MSC_VER` is only defined when building with Visual Studio. I've changed most of the `_MSC_VER` checks to `_WIN32`.